docs/modules/ROOT/pages/how-tos/logging/increase-elasticsearch-storage-size.adoc

```diff
@@ -0,0 +1,283 @@
+= Increase Storage Size of an OCP Logging Elasticsearch instance
+
+[abstract]
+This page describes how to increase the underlying storage size of the OpenShift Cluster Logging Elasticsearch instance.
+
+== Starting situation
+
+* You already have an OpenShift 4 with Cluster Logging enabled.
+* The Cluster Logging instance is managed and is of type `elasticsearch`.
+* You have admin-level access to the cluster.
+* You want to increase the storage size of the Elasticsearch cluster.
+
+== Prerequisites
+
+* `kubectl`
+* `curl`
+* `jq`
+* `yq` https://mikefarah.gitbook.io/yq[yq YAML processor] (version 4 or higher)
+* `commodore`, see https://syn.tools/commodore/running-commodore.html[Running Commodore]
+
+== Prepare local environment
+
+. Configure API access
++
+[source,bash]
+----
+export COMMODORE_API_URL=https://api.syn.vshn.net <1>
+export COMMODORE_API_TOKEN=<your lieutenant token>
+
+# Set Project Syn cluster and tenant ID
+export CLUSTER_ID=<lieutenant-cluster-id> # Looks like: c-cluster-id-1234
+export TENANT_ID=$(curl -sH "Authorization: Bearer ${COMMODORE_API_TOKEN}" ${COMMODORE_API_URL}/clusters/${CLUSTER_ID} | jq -r .tenant)
+----
+<1> Replace with the API URL of the desired Lieutenant instance.
+
+. Create a local directory to work in and compile the cluster catalog
++
+[source,bash]
+----
+export WORK_DIR=/path/to/work/dir
+mkdir -p "${WORK_DIR}"
+pushd "${WORK_DIR}"
+
+commodore catalog compile "${CLUSTER_ID}"
+----
++
+[TIP]
+====
+We strongly recommend creating an empty directory, unless you already have a work directory for the cluster you're about to work on.
+This guide will run Commodore in the directory created in this step.
+====
+
+== Increase PVC sizes
+
+=== Update Catalog
+
+. Set desired size
++
+[source,bash]
+----
+STORAGE_SIZE=250Gi <1>
+----
+<1> Replace with the desired PVC size.
+
+. Update Commodore catalog
++
+[source,bash]
+----
+pushd "inventory/classes/${TENANT_ID}/"
+
+yq eval -i "parameters.openshift4_logging.clusterLogging.logStore.elasticsearch.storage.size += \"${STORAGE_SIZE}\"" \
+  ${CLUSTER_ID}.yml
+
+git commit -a -m "Set Elasticsearch backing storage to \"${STORAGE_SIZE}\" on ${CLUSTER_ID}"
+git push
+popd
+----
++
+. Compile catalog
++
+[source,bash]
+----
+commodore catalog compile ${CLUSTER_ID} --push -i
+----
```

Comment:
Maybe this can also be combined with https://syn.tools/syn/how-tos/change-parameter.html